src/QueryGrammar.php

```diff
@@ -1038,10 +1038,13 @@ public function compileInsert(Builder $builder, array $values): array
                 '_id'    => $doc['id'],
             ];
 
+            if ($routing = $builder->getRouting()) {
```

Comment:
Wanted to ask, opinion on whether or not this should be an `elseif()`?

Reply:
I think this is good - I think it's possible to have different values for both `_parent` and `_routing` in ES so this should work well.